gpio_test/gpio_config/gpio_config_builder.py

```diff
@@ -149,6 +152,10 @@ def correct_dd_holds(stream, bpos):
         stream_l = build_stream_independent(stream_l, config_l[k])
     elif gpio_l[k][1] == H_SPECIAL:
         stream_l = build_stream_special(stream_l, config_l[k])
+    elif gpio_l[k][1] == H_NONE: #generalise this may be by using NUM_IO and knowing which pin is faulty and just hard coding that pin's stream based on the violation type
```

Comment:
"Hardcoding" is fine here since this applies only to the H_NONE pins. The comment can therefore be removed.